src/Providers/UserProviders.jsx

```diff
@@ -16,12 +20,21 @@ class UserProvider extends Component {
                 const user = await generateUserDocument(userAuth);
                 this.setState({ user });
             }
+            // TODO: in the future do an authentication route that returns name and email of user?
```

Comment:
If we decide to implement this route in the future, it can also lead to the removal of `user` and the generateUserDocument logic.

However, it is not necessary.